dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -381,6 +382,24 @@ public class ConnectorArguments extends DefaultArguments {
           .ofType(Integer.class)
           .defaultsTo(OPT_THREAD_POOL_SIZE_DEFAULT);
 
+  // Kerberos principal
+  private final OptionSpec<String> optionKerberosPrincipal =
+      parser
+          .accepts(
+              OPT_KERBEROS_PRINCIPAL,
+              "Set Kerberos principal. It is usually a Service principal of the form: "
+                  + "DUMPER/webserver.example.com@EXAMPLE.COM . (No default value)")
+          .withOptionalArg()
```

Comment:
If there's no default, shouldn't this be `withRequiredArgs` as the keytab option below?

Reply:
Right!
Fixed
Thanks!